components/arm/xarm/xarm.go

```diff
@@ -56,57 +56,50 @@ var xArm6modeljson []byte
 //go:embed xarm7_kinematics.json
 var xArm7modeljson []byte
 
-// ModelName6DOF is a function used to get the string used to refer to the xarm model of 6 dof.
-var ModelName6DOF = resource.NewDefaultModel("xArm6")
+//go:embed xarmlite_kinematics.json
+var xArmLitemodeljson []byte
 
-// ModelName7DOF is a function used to get the string used to refer to the xarm model of 7 dof.
-var ModelName7DOF = resource.NewDefaultModel("xArm7")
+const (
+	ModelName6DOF = "xArm6"    // ModelName6DOF is the name of an xArm6
+	ModelName7DOF = "xArm7"    // ModelName7DOF is the name of an xArm7
+	ModelNameLite = "xArmLite" // ModelNameLite is the name of an xArmLite
+)
 
 // Model returns the kinematics model of the xarm arm, also has all Frame information.
-func Model(name string, dof int) (referenceframe.Model, error) {
-	switch dof {
-	case 6:
+func Model(name, modelName string) (referenceframe.Model, error) {
+	switch modelName {
+	case ModelName6DOF:
 		return referenceframe.UnmarshalModelJSON(xArm6modeljson, name)
-	case 7:
+	case ModelNameLite:
+		return referenceframe.UnmarshalModelJSON(xArmLitemodeljson, name)
+	case ModelName7DOF:
 		return referenceframe.UnmarshalModelJSON(xArm7modeljson, name)
 	default:
 		return nil, errors.New("no kinematics model for xarm with specified degrees of freedom")
```

Comment:
[nit] Would prefer this error be updated to track the change to switching based on a model identifier and not arm DoF

Reply:
Updated